# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kvm/always-compatfd.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For old glibc's one might have to disable AIO, however KVM always
requires compatfd to be built.

  - Rene Rebe <rene@exactcode.de>

--- qemu/Makefile.vanilla	2009-02-04 20:37:40.000000000 +0000
+++ qemu/Makefile	2009-02-04 20:38:03.000000000 +0000
@@ -62,6 +62,10 @@
 
 ifdef CONFIG_AIO
 BLOCK_OBJS += compatfd.o
+else
+ifeq ($(USE_KVM), 1)
+BLOCK_OBJS += compatfd.o
+endif
 endif
 
 ######################################################################
--- qemu/Makefile.target.vanilla	2009-02-04 20:39:02.000000000 +0000
+++ qemu/Makefile.target	2009-02-04 20:39:16.000000000 +0000
@@ -622,6 +622,10 @@
 
 ifdef CONFIG_AIO
 OBJS+=compatfd.o
+else
+ifeq ($(USE_KVM), 1)
+OBJS+=compatfd.o
+endif
 endif
 
 LIBS+=-lz
